### DISCUSSION TITLE
part-ii/2021-70/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#324] [Question comments]
Part II 2021 Galois Theory: Paper 3, Section II, 18I 

https://questions.tripos.org/part-ii/2021-70/

**Comment by DannyDuberstein** [comment]
For last bit write as poly in p,q with 2 coefficients to be determined. Set p = 0 to recover one coefficient. For the other, use that discriminant vanishes iff gcd(f,f') nontrivial, do Euclid explicitly
